index.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+require_once 'functions.php';
+require_once 'dbConnect.php';
+
+$db = connectdb();
+
+
+$shinkansens = fetchData($db);
+
+$display = '';
+foreach($shinkansens as $shinkansen) {
```

Comment:
move the foreach to be inside the display function to increase the amount of testable code